corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -128,26 +128,28 @@
     {% endblocktrans %}
 </div>
 {% endif %}
-<div class="pull-right">
-    {% if auth_context.authenticated %}
-        <i class="icon-lock"></i>
-        {% if user_info.id != auth_user_info.id %}
-        {% blocktrans with auth_user=auth_user_info|pretty_doc_info user=user_info|pretty_doc_info %}
-            Submitted by {{ auth_user }} on behalf of {{ user }}
-        {% endblocktrans %}
+<div class="clearfix">
+    <div class="pull-right">
```

Comment:
@biyeun when i move this below it kind of floats inside the tab and looks weird. also getting rid of the wrapping div doesn't work.